src/main/java/org/wikiclean/WikiClean.java

```diff
@@ -131,7 +131,13 @@ public String getWikiMarkup(String s) {
       return "";
     }
 
-    return s.substring(textStart + 27, textEnd);
+    String s2 = s.substring(textStart + 27, textEnd);
```

Comment:
The XML files exported from https://en.wikipedia.org/wiki/Special:Export have an additional attribute `bytes` in `text` element. Need to ignore this as well while reading wiki-markup.